File: DevExpress-Demo/VB/MVVMDemo.Wpf/Modules/DXBinding/RegistrationHelper.vb
Namespace MVVMDemo.DXBindingDemo

    Public Module RegistrationHelper

        Public Function CanRegister(ByVal userName As String, ByVal acceptTerms As Boolean) As Boolean
            Return Not String.IsNullOrEmpty(userName) AndAlso acceptTerms
        End Function
    End Module
End Namespace

File: MyDX-Demo/MyDX-Demo/Module/MVVM-Demo/DXBindingDemo/RegistrationHelper.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MyDX_Demo.Module.MVVM_Demo.DXBindingDemo
{
    public class RegistrationHelper
    {
        public static bool CanRegister(string userName, bool acceptTerms)
        {
            return !string.IsNullOrEmpty(userName) && acceptTerms;
        }
    }
}